numba/jitclass/decorators.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import, print_function
+from numba import types
+from .base import register_class_type, ClassBuilder
+from .immutable import ImmutableClassBuilder
+
+
+def jitclass(spec, immutable=False):
+    """
+    A decorator for creating a jitclass.
+
+    Args
+    ----
+    spec:
+        Specifies the types of each field on this class.
+        Must be a dictionary or a sequence.
+        With a dictionary, use collections.OrderedDict for stable ordering.
+        With a sequence, it must contain 2-tuples of (fieldname, fieldtype).
+
+    immutable:
```

Comment:
Since immutable isn't implemented yet, you should be sure to raise an error if set (or simply remove it from the signature).